modules/grizzly/src/main/java/org/glassfish/grizzly/ssl/HandshakeListener.java

```diff
@@ -1,12 +0,0 @@
-package org.glassfish.grizzly.ssl;
```

Comment:
Cause of the TCK failure. Tyrus depends on this interface, so it must not be moved outside the class. Breaking change.